tools/gsc_high_dimensions_visualisation/high_dim_visu.xml

```diff
@@ -1,12 +1,12 @@
 <tool id="high_dimensions_visualisation" name="Generate PCA, tSNE and HCPC" version="0.9.3">
     <description>from highly dimensional expression data</description>
     <requirements>
-        <requirement type="package" version="1.3.2=r3.3.2_0">r-optparse</requirement>
-        <requirement type="package" version="1.39=r3.3.2_0">r-factominer</requirement>
-        <requirement type="package" version="1.0.5=r3.3.2_0">r-factoextra</requirement>
-        <requirement type="package" version="0.13=r3.3.2_0">r-rtsne</requirement>
-        <requirement type="package" version="2.2.1=r3.3.2_0">r-ggplot2</requirement>
-        <requirement type="package" version="0.4.1=r3.3.2_0">r-ggfortify</requirement>
+        <requirement type="package" version="1.6.2=r35h6115d3f_0">r-optparse</requirement>
+        <requirement type="package" version="1.41=r351h6115d3f_1">r-factominer</requirement>
+        <requirement type="package" version="1.0.5=r351_1">r-factoextra</requirement>
+        <requirement type="package" version="0.15=r351he1b5a44_0">r-rtsne</requirement>
```

Comment:
@bellenger-l note that with this new r-rtsne version, the option `normalize` seems to be back !!!!

Reply:
Done !